webview/src/plots/components/Plots.tsx

```diff
@@ -54,20 +49,10 @@ const PlotsContent = () => {
     return <EmptyState>Loading Plots...</EmptyState>
   }
 
-  if (
-    !hasCheckpointData &&
-    !hasComparisonData &&
-    !hasTemplateData &&
-    !hasCustomData
-  ) {
+  if (!hasCheckpointData && !hasComparisonData && !hasTemplateData) {
```

Comment:
Explanation of the new walkthrough in more detail:
* has plots render the get started screen if there are no selected plots or experiments even if there is custom plot data 
* has the custom section render "No Data to Display" if there is no data or there are no experiments
* has the custom section render "No Plots Added" if there is experiment data but no plots added

Reply:
> has plots render the get started screen if there are no selected plots or experiments even if there is custom plot data

[Q] Does this mean that we hide added custom plots even though there is something to display? How about we show a non-fullscreen `<EmptyState>` (`isFullScreen=false`) version of the get started screen above the custom plots? What would that look like?

Reply:
> [Q] Does this mean that we hide added custom plots even though there is something to display? How about we show a non-fullscreen <EmptyState> (isFullScreen=false) version of the get started screen above the custom plots? What would that look like?


Yes, we do hide it since I was thinking it would look odd but now that actually it looks pretty good! I think the first one looks better but does require a little more CSS:


<img width="1792" alt="Screenshot 2023-03-03 at 4 58 27 PM" src="https://user-images.githubusercontent.com/43496356/222850511-317b7942-3fe1-4bbc-826e-c25e056bd8a4.png">

<img width="1792" alt="Screenshot 2023-03-03 at 5 04 14 PM" src="https://user-images.githubusercontent.com/43496356/222851745-b58ed8c3-2582-41af-9ddb-2a7746c99522.png">

If we want to keep this, I'm assuming we'd need to adjust the message since there are actually some plots to display... Maybe "There are no selected plots or experiments"? What does everyone else think?

Reply:
I definitely think this is better and we should not be hiding anything that has been selected based on some other hidden condition. 

> Maybe "There are no selected plots or experiments"

👍🏻. I think there are some variations on this statement but that works. 

I also think we should be displaying all commit data in custom plots as well as experiments.

Reply:
> I also think we should be displaying all commit data in custom plots as well as experiments.

Added a checkbox in the original ticket 🙏🏻.